kong/pdk/service/response.lua

```diff
@@ -198,6 +198,8 @@ local function new(pdk, major_version)
   --   kong.log.inspect(headers.x_another[1])    -- "foo bar"
   --   kong.log.inspect(headers["X-Another"][2]) -- "baz"
   -- end
+  -- Note that this function returns a proxy table
+  -- which cannot be iterated with pairs or used as oprand of #.
```

Comment:
Is there a particular reason for this? Should we rather fix this to allow `#` etc.? Aka implement those metamethods?

Reply:
I've created a fix for that, but it's incomplete. After a discussion, we found this may take more time than expected and is not prioritized. So as a temporary solution, we choose to document it to avoid further confusion.